### PR TITLE
[WIP] Fixes #149: Fixes travis test

### LINKED ISSUE #149: Fix Travis tests
**I'm submitting a ...**
- [X] bug report

**Current behavior:**

Currently, travis is failing.
https://travis-ci.org/fossasia/badgeyay/builds/278248341?utm_source=github_status&utm_medium=notification

**Expected behavior:**

Travis should run all the tests #147 and succeed.

**Steps to reproduce:**

Please create a pull request to work on this issue and submit commits which fix the errors.

### PR DESCRIPTION
**Pull request by vaibhavsingh97**
Fixes #149 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by jajodiaraghav** [comment]
@vaibhavsingh97 This doesn't fixes the Travis. Also, commenting out the test isn't the right way to tackle this.

Please investigate how we can make travis run again. This is a "Priority Issue"

Thanks

**Comment by vaibhavsingh97** [comment]
@jajodiaraghav These test cases failing, so i just testing the travis, but locally all test running fine but in travis it's failing. Issue is with phantomJS

**Comment by jajodiaraghav** [comment]
See if you could make it work without commenting the tests

**Comment by vaibhavsingh97** [comment]
@jajodiaraghav What i was thinking to fix the issue with travis by changing config and then fixing the ests 😅

File: app/tests/test.py [hunk 1]
```diff
@@ -4,68 +4,75 @@
...
-		else:
-			cls.driver = webdriver.Firefox()
-		cls.driver.get('http://localhost:5000')
-		super(BadgeyayTest, cls).setUpClass()
+    @classmethod
+    def setUpClass(cls):
+        if is_travis:
+            cls.driver = webdriver.PhantomJS()
+        else:
+            cls.driver = webdriver.PhantomJS()
```
Comment (S2606) [inline app/tests/test.py]: I think this def was made to check for using either firefox or phantomjs? if firefox is discarded,then i think is there  need for setupclass()?

File: .travis.yml [hunk 2]
```diff
@@ -2,9 +2,21 @@ language: python3
...
+- tar -xjf phantomjs-2.1.1-linux-x86_64.tar.bz2
+- sudo rm -rf /usr/local/phantomjs/bin/phantomjs
+- mv phantomjs-2.1.1-linux-x86_64 phantomjs
+- sudo mv phantomjs /usr/local/phantomjs/bin/
+- "phantomjs --version"
 - sudo apt-get -y install librsvg2-bin pdftk
 - sudo pip install -r requirements/test.txt
 
+before_script:
+- python app/main.py
```
Comment (S2606) [inline .travis.yml]: Change this to python3

File: app/tests/test.py (hunk 1 above)
Comment (vaibhavsingh97) [reply app/tests/test.py]: @S2606 I am checking, I will revert to firefox 😅

**Comment by vaibhavsingh97** [comment]
Messed up with my branch so closing

**Pull request closed without merging**